tests/stdlib/trandom.nim

```diff
@@ -36,24 +33,33 @@ proc main =
   doAssert rand(0) == 0
   doAssert sample("a") == 'a'
 
-  when compileOption("rangeChecks"):
```

Comment:
`doAssertRaises` is good but keep `when compileOption("rangeChecks"):`, otherwise you get `SIGFPE: Arithmetic error.` when running with -d:danger for example